scripts/build

```diff
@@ -1,5 +1,4 @@
 #!/usr/bin/env bash
-set -e
```

Comment:
Maybe, instead i should figure out how to swallow the error with the dotnet tool command.